service/chartconfig/v1/appr/appr.go

```diff
@@ -84,10 +91,26 @@ func (c *Client) GetReleaseVersion(name, channel string) (string, error) {
 }
 
 // PullChartTarball downloads a tarball with the chart described by the given
-// custom object, returning the file path.
-// TODO
+// chart name and channel, returning the file path.
 func (c *Client) PullChartTarball(name, channel string) (string, error) {
-	return "", nil
+	release, err := c.GetReleaseVersion(name, channel)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	p := path.Join("packages", c.organization, name, release, "helm", "pull")
+
+	req, err := c.newRequest("GET", p)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	chartTarball, err := c.doFile(req)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	return chartTarball, nil
```

Comment:
The var indicates it is the tarball itself but it is only a file path.

Reply:
Good point, renamed to make clear it's a path.